WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabItemsStore.swift

```diff
@@ -0,0 +1,91 @@
+import WordPressFlux
+
+
+class ReaderTabItemsStore: Observable {
+
+    let changeDispatcher = Dispatcher<Void>()
+
+    enum State {
+        case loading
+        case ready([ReaderTabItem])
+        case error(Error)
+
+        var isLoading: Bool {
+            switch self {
+            case .loading:
+                return true
+            case .error, .ready:
+                return false
+            }
+        }
+    }
+
+    var state: State = .ready([]) {
```

Comment:
Shouldn't `state` start out loading?

Reply:
The loading is requested by the view, so at the init time, the store it's not loading, and it needs to be set to '.loading' only when it's actually loading, to prevent multiple calls to getItems(), so I'd leave it as it is.